e2e/support/helpers/component-testing-sdk/component-embedding-sdk-helpers.tsx

```diff
@@ -60,15 +61,18 @@ export function mountSdkContent(
   cy.intercept("GET", "/api/user/current").as("getUser");
 
   cy.mount(
-    <MetabaseProvider
-      {...sdkProviderProps}
-      authConfig={{
-        ...DEFAULT_SDK_AUTH_PROVIDER_CONFIG,
-        ...sdkProviderProps?.authConfig,
-      }}
-    >
-      {children}
-    </MetabaseProvider>,
+    <React.StrictMode>
```

Comment:
Adds StrictMode to the render tree of the SDK component tests, so we can detect React errors that only happens in strict mode.

Reply:
Should we put it behind a flag?
I see strict mode as a different env/dimension, I'm not sure we should run all tests by default on it as it's not enabled on production

Reply:
@npretto Yes, applying strict mode to only some test sounds good. It would be good to know that those bugs are caused by strict mode behavior, and not a general prod bug.

Reply:
fixed in https://github.com/metabase/metabase/pull/52885/commits/453bd97376b9fb612af481882920ce179fbce256